src/EventStore.Client.Streams/MaximumAppendSizeExceededException.cs

```diff
@@ -6,13 +6,29 @@ namespace EventStore.Client {
 	/// Exception thrown when an append exceeds the maximum size set by the server.
 	/// </summary>
 	public class MaximumAppendSizeExceededException : Exception {
+		/// <summary>
+		/// The configured maximum append size.
+		/// </summary>
+		public uint MaxAppendSize { get; }
+
 		/// <summary>
 		/// Constructs a new <see cref="MaximumAppendSizeExceededException"/>.
 		/// </summary>
 		/// <param name="maxAppendSize"></param>
 		/// <param name="innerException"></param>
-		public MaximumAppendSizeExceededException(int maxAppendSize, Exception? innerException) :
+		public MaximumAppendSizeExceededException(uint maxAppendSize, Exception? innerException = null) :
```

Comment:
When I get this exception on the client side it shows the `maxAppendSize` as `0`.

```
Unhandled exception. EventStore.Client.MaximumAppendSizeExceededException: Maximum Append Size of 0 Exceeded.
   at EventStore.Client.Streams.BatchAppendResp.ToWriteResult()
```

Reply:
it wasn't being sent from the server, that's fixed now